backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -825,13 +851,14 @@ void addTestResult_savesResults() {
     facility.addDefaultDeviceSpecimen(devA);
 
     // WHEN
+    List<MultiplexResultInput> testResultInput = makeMultiplexTestResult();
     AddTestResultResponse res =
-        _service.addTestResult(
-            devA.getInternalId(), TestResult.POSITIVE, patient.getInternalId(), null);
+        _service.addMultiplexResult(
+            devA.getInternalId(), testResultInput, patient.getInternalId(), null);
 
     // THEN
     List<Result> results = _resultRepository.findAllByTestOrder(res.getTestOrder());
-    assertEquals(1, results.size());
+    assertEquals(3, results.size());
```

Comment:
Ditto to the [above comment.](https://github.com/CDCgov/prime-simplereport/pull/4760/files#r1029726095)